src/VisualStudio/Core/Def/Implementation/TaskList/ProjectExternalErrorReporter.cs

```diff
@@ -312,7 +346,8 @@ public int ClearErrors()
                     mappedStartColumn: mappedStartColumn,
                     mappedEndLine: mappedEndLine,
                     mappedEndColumn: mappedEndColumn),
-                language: language);
+                language: language,
+                helpLink: helpLink);
         }
 
         private static bool IsCompilerDiagnostic(string errorId)
```

Comment:
A bit unrelated to the changes: This method does `errorId.Substring(0, 2)` just to compare this to "CS" and "VB"

`StartsWith` can be faster, though I don't know if this can have any noticeable impact.